pkg/pipeline/remote/bitbucketserver/client.go

```diff
@@ -327,7 +327,6 @@ func (c *client) SetPipelineFileInRepo(repoURL string, branch string, accessToke
 	}
 	if currentContent != nil {
 		//update pipeline file
-		message = fmt.Sprintf("Update %s file", currentFileName)
```

Comment:
I am not sure how important this statement is for debug purposes, but we can keep this and update values map with message